asciidoc/getting_started/cypress/plugin-information.adoc

```diff
@@ -0,0 +1,57 @@
+= @ensono-stacks/cypress
+:sidebar_label: @ensono-stacks/cypress
+:description: Using cypress with Ensono Stacks!
+:keywords: Nx, monorepo, stacks, ensono, cypress, generator, executor
+
+Using the `@ensono-stacks/cypress` plugin can help you get started with testing using Cypress. Built upon the link:https://nx.dev/packages/cypress[@nx/cypress] plugin, Ensono Stacks provides additional capabilities to help you hit the ground running!
+
+Additionally, Cypress tests can be added to your deployment pipelines through the various deployment generators.
+
+[NOTE]
+====
+Visit the link:../../testing/testing_in_nx/frontend_testing_in_nx.md[Frontend Testing in an Nx monorepo] and link:../../testing/testing_in_nx/cypress_nx.md[Testing with Cypress] user guides for more information!
+====
+
+== Setting up @ensono-stacks/cypress
+
+Install the `@ensono-stacks/cypress` plugin with the following command:
+
+=== npm
+
+[source]
```

Comment:
Do we need `[source]` when not providing a language?
Could we just use ----?

Reply:
removed the [source] tag